jlc_kicad_tools/jlc_lib/cpl_fix_rotations.py

```diff
@@ -59,46 +55,22 @@ def FixRotations(input_filename, output_filename, db):
             package_index = i
           elif row[i] == "Rot":
             rotation_index = i
-          elif row[i] == "PosX":
-            posx_index = i
-          elif row[i] == "Side":
-            side_index = i
         if package_index is None:
-          _LOGGER.logger.warning("Failed to find 'Package' column in the csv file")
+          logging.warning("Failed to find 'Package' column in the csv file")
           return False
         if rotation_index is None:
-          _LOGGER.logger.warning("Failed to find 'Rot' column in the csv file")
-          return False
-        if side_index is None:
-          _LOGGER.logger.warning("Failed to find 'Side' column in the csv file")
-          return False
-        if posx_index is None:
-          _LOGGER.logger.warning("Failed to find 'PosX' column in the csv file")
+          logging.warning("Failed to find 'Rot' column in the csv file")
           return False
         # Replace column names with labels JLC wants.
         for i in range(len(row)):
           if row[i] in HEADER_REPLACEMENT_TABLE:
             row[i] = HEADER_REPLACEMENT_TABLE[row[i]]
       else:
-        rotation = float(row[rotation_index])
```

Comment:
I assume this change and the one below are unintentional?

Reply:
Sorry, I did not change that file. git log shows e2e447e73027acd21f1bdbec3e447fc0506f218b as last change of that file,